index.html

```diff
@@ -36,5 +36,135 @@ <h3>LET'S CONNECT</h3>
+             </ul>
+        <button type="submit" class="project-button">See Project</button>
+       </div>
+</div>       
+<div class="grid-container2">
+        <div class="grid-item1">
+        <img src="Snapshoot Portfolio(2).png" alt="portfolio"></div>
+        <div class="grid-item2">
+        <h2>Multi-Post <br>Stories</h2>
+            <ul>
+                <li style="list-style: none;" id="canopy">CANOPY</li>
+                <li>Back End Dev</li>
+                <li>2015</li>
+            </ul>
+            <p>A daily selection of privately<br>
+                 personalized reads; no accounts or<br>
+                  sign-ups required.</p>
+                  <div class="button-container">
+                <ul style="list-style: none;" class="language-button">
+               <button type="submit"><li>html</li></button> 
+             <button type="submit"><li>css</li></button>
+               <button type="submit"><li>javascript</li></button>
+            </ul>  
+        </div>
+            <button type="submit" class="project-button">See Project</button>
+        </div>
+</div>        
+<div class="grid-container3"> 
+            <div class="grid-item1"><img src="Snapshoot Portfolio(3).png" alt="portfolio"></div>
+            <div class="grid-item2">
+        <h2>Tonic</h2>
+            <ul >
+                <li style="list-style: none;" id="canopy">CANOPY</li>
+                <li>Back End Dev</li>
+                <li>2015</li>
+            </ul>
+            <p>A daily selection of privately<br>
+                 personalized reads; no accounts or<br>
+                  sign-ups required.</p>
+                  <ul style="list-style: none;" class="language-button">
+                    <button type="submit"><li>html</li></button> 
+                  <button type="submit"><li>css</li></button>
+                    <button type="submit"><li>javascript</li></button>
+                 </ul>  
+            <button type="submit" class="project-button">See Project</button>
+         </div>
+</div>         
+<div class="grid-container4"> 
+           <div class="grid-item1"><img src="Snapshoot Portfolio(4).png" alt="portfolio"></div>
+                 <div class="grid-item2"><h2>Multi-Post <br>Stories</h2>
+            <ul>
+                <li style="list-style: none;" id="canopy">CANOPY</li>
+                <li>Back End Dev</li>
+                <li>2015</li>
+            </ul>
+            <p>A daily selection of privately<br>
+                 personalized reads; no accounts or<br>
+                  sign-ups required.</p>
+                  <ul style="list-style: none;" class="language-button">
+                    <button type="submit"><li>html</li></button> 
+                  <button type="submit"><li>css</li></button>
+                    <button type="submit"><li>javascript</li></button>
+                 </ul>  
+            <button type="submit" class="project-button">See Project</button> 
+    </div>
+</div>    
```

Comment:
- To improve the HTML structure of your site and to make it more maintainable, I'd suggest you wrap all your project cards in a `section` tag and align them with a `grid` display as required in the project. 🙏 
- According to the project requirements, please note that the content of your project card should be aligned with `flex` display. Kindly fix this issue 🙏